quipucords/cli/tests.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2017 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+#
+"""Test the CLI module"""
+
+import contextlib
+import unittest
+import sys
+import six
+from cli.cli import CLI
+
+
+# pylint: disable=too-few-public-methods
+class HushUpStderr(object):
```

Comment:
Let's move `HushUpStderr` and `redirect_stdout` to a common test utilities library.

Reply:
done